app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpViewModel.kt

```diff
@@ -94,7 +63,7 @@ class NextUpViewModel(
 			else -> null
 		}
 		val logo = apiClient.GetLogoImageUrl(item, ImageOptions())
-		val title = item.getTitle()
+		val title = BaseItemUtils.getDisplayName(context, item)
```

Comment:
You can keep the extension function in the util class so it becomes `item.getDisplayName(context)`

Reply:
In the most recent push I also made the rest of the functions extension functions, but I can change them back if they shouldn't be.